src/tests/topp/ProteinQuantifier_9_output_mztab.mzTab

```diff
@@ -6,16 +6,19 @@ MTD	description	OpenMS export from consensusXML
 MTD	protein_search_engine_score[1]	[, , one-peptide-rule, ]
 MTD	peptide_search_engine_score[1]	[, , Mascot, ]
 MTD	psm_search_engine_score[1]	[, , Mascot, ]
-MTD	software[1]	[MS, MS:1000752, TOPP software, 2.6.0-pre-feature-MzTab-stream-2020-05-04]
+MTD	software[1]	[MS, MS:1000752, TOPP software, 2.6.0-pre-fix-mzTabMTD-2020-05-12]
 MTD	software[2]	[, , Mascot, ]
-MTD	software[2]-setting[1]	null
-MTD	software[2]-setting[2]	null
-MTD	software[2]-setting[3]	null
-MTD	software[2]-setting[4]	fragment_mass_tolerance:0.0
-MTD	software[2]-setting[5]	fragment_mass_tolerance_unit:Da
-MTD	software[2]-setting[6]	precursor_mass_tolerance:0.0
-MTD	software[2]-setting[7]	precursor_mass_tolerance_unit:Da
-MTD	software[2]-setting[8]	enzyme:unknown_enzyme
+MTD	software[2]-setting[1]	db:
```

Comment:
should we print out empty settings at all? if empty use null?

Reply:
But if it is just says "null" no one knows what parameter was "null". That was my point.

Reply:
db:null ?

Reply:
Ok that is fine for me.

Reply:
it's more a question than a suggestion